src/_P021_Level.ino

```diff
@@ -187,32 +207,27 @@ boolean Plugin_021(uint8_t function, struct EventStruct *event, String& string)
       P021_CHECK_VALUE      = -1;                  // No input source assigned
       P021_SETPOINT         = 25.0f;               // Switch output active above 25 [deg C]
       P021_HYSTERESIS       = 5.0f;                // Switch off hysteresis 5 [deg C] below switch on value
-      P021_MIN_TIME         = 30;                  // Once switched on output should be active at least 30 [min]
-      P021_INTERVAL_TIME    = 24;                  // Output shall run after 24 [hour] stand still
-      P021_FORCE_TIME       = 5;                   // Forced circulation for 5 [min]
+      P021_MIN_TIME         = minutes2seconds(30); // Once switched on output should be active at least 30 [min]
+      P021_INTERVAL_TIME    = hours2seconds(24);   // Output shall run after 24 [hour] stand still
+      P021_FORCE_TIME       = minutes2seconds(5);  // Forced circulation for 5 [min]
       P021_OPMODE           = P021_OPMODE_OFF;     // Don't touch output unless selected by operator
       P021_FLAGS            = 0;                   // Reset all flags
       UserVar.setFloat(event->TaskIndex, P021_VALUE_STATE, (float)P021_STATE_IDLE);
-      P021_remote[event->TaskIndex]    = 0;        // Remote control state is "off"
-      P021_timestamp[event->TaskIndex] = millis(); // Start counting time from now on
+      P021_remote[event->TaskIndex] = 0;           // Remote control state is "off"
       break;
     }
 
     case PLUGIN_WEBFORM_LOAD:
     {
        #ifdef PLUGIN_021_DEBUG
       {
-        P021_control_state control_state = (P021_control_state)UserVar.getFloat(event->TaskIndex, P021_VALUE_STATE);
+        const P021_control_state control_state = (P021_control_state)UserVar.getFloat(event->TaskIndex, P021_VALUE_STATE);
 
         // Add some debug information
-        String msg = F("State= ");
-        msg += P021_printControlState(control_state);
-        msg += (F(", Pump= "));
-        msg += (int)UserVar.getFloat(event->TaskIndex, P021_VALUE_OUTPUT) == 1 ? F("on") : F("off");
-        msg += F(", Remote= ");
-        msg += P021_remote[event->TaskIndex];
-        msg += F(", Timer= ");
-        msg += (int)(millis2seconds(timePassedSince(P021_timestamp[event->TaskIndex])));
+        String msg = strformat(F(", State= %s "), P021_printControlState(control_state));
+        msg += strformat(F(", Output= %s"), (int)UserVar.getFloat(event->TaskIndex, P021_VALUE_OUTPUT) == 1 ? F("on") : F("off"));
```

Comment:
While `strformat()` works fine processing a `F("")` string into the %s placeholder on ESP32, that's not guaranteed to always work on an ESP8266, so a wrapper should be applied like `String(<condition> ? F("x") : F("y")).c_str()` (or on a single F("") string where applicable).